### PR TITLE
Add codemodel recipe

### PR DESCRIPTION
**Pull request by dwhswenson**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dwhswenson** [comment]
@conda-forge/help-python ready for review!

**Comment by npavlovikj** [comment]
@dwhswenson , the recipe looks good to me. I would just remove the `#doc_url` line, and since you are the developer of the package, you can put the LICENSE file in the pypi tarball, instead of adding it separately. 
If this is something you can quickly update, I can merge the recipe afterwards.

**Comment by dwhswenson** [comment]
@npavlovikj : This should be ready now!

I cut release 0.0.5, which to allowed me to:

* add `LICENSE` to the tarball (removed from here)
* put a stub for docs on RTD. (I'd previously left that field commented to remind me to add it later.)

File: recipes/codemodel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - astor
```
Comment (xhochy) [inline recipes/codemodel/meta.yaml]: The setup.py only lists `astor`, why is here also `black` and `isort` listed? https://github.com/dwhswenson/codemodel/blob/master/setup.py#L23

File: recipes/codemodel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - black
+    - isort
+
+test:
+  imports:
+    - codemodel
+    - codemodel.tests
```
Comment (xhochy) [inline recipes/codemodel/meta.yaml]: Only test the main module here, tests don't need to be import-tested and most often shouldn't either as they have additional requirements that are not necessary for running the main code.
```suggestion
```

File: recipes/codemodel/meta.yaml (hunk 1 above)
Comment (dwhswenson) [reply recipes/codemodel/meta.yaml]: Had drafted the recipe based on a branch in progress. Removed these (they'll come back in a future release!)

**Pull request merged**